Makefile

```diff
@@ -1,9 +1,11 @@
 SH_FILES := $(shell find . -type f -name "*.sh" ! -path "./node_modules/*" ! -path "./.venv/*" ! -path "./testCourse/*")
 
+.PHONY: build
```

Comment:
on my machine, running `make build` doesn't work without this.

Reply:
https://stackoverflow.com/a/59558099/5684541

> It happens when you have a file with the same name as Makefile target name in the directory where the Makefile is present.

Reply:
https://stackoverflow.com/a/59558099/5684541

> It happens when you have a file with the same name as Makefile target name in the directory where the Makefile is present.

Reply:
Why do you have a `build` file/directory at the root of your repo?

Reply:
Very good question! Not sure when that got there, or how, but it was empty, so I just deleted it and it now works fine. Fixed in [44649dc](https://github.com/PrairieLearn/PrairieLearn/pull/11501/commits/44649dc5e039ea831674c71743e8261ea94282e6). Thanks for the sanity check.